buildSrc/src/main/java/Dependencies.kt

```diff
@@ -27,6 +28,9 @@ private object Versions {
     const val tools_test_rules = "1.1.0"
     const val tools_test_runner = "1.1.0"
     const val uiautomator = "2.2.0"
+    const val robolectric = "4.3"
+    const val megazord = "0.53.0"
```

Comment:
This used to be problematic if the version here and the version used by AC start to differ. Afaik Fenix no longer does this either? If we need it then we should reference the `support-test-appservices` component which will import the right version as transitive `api` dependency.